cgra/util.py

```diff
@@ -39,7 +39,7 @@ def create_cgra(width: int, height: int, io_sides: IOSide,
                 tile_id_width: int = 16,
                 num_tracks: int = 5,
                 add_pd: bool = True,
-                use_sram_stub: bool = True,
+                use_sim_sram: bool = True,
```

Comment:
this flag change will require a change to the mflowgen RTL node. I can make push it if you don't mind.

Reply:
pushed the relevant change